src/api/users.js

```diff
@@ -24,7 +24,8 @@ export async function authenticate (ctx, email) {
   email = unescape(email)
 
   try {
-    const user = await UserModelAPI.findOne({email}).exec()
+    const emailReg = new RegExp(email, 'i')
```

Comment:
Escape the regular expression with [escapeRegExp](https://lodash.com/docs/4.17.4#escapeRegExp)